js/ui/widget/ui.widget.d.ts

```diff
@@ -137,7 +138,7 @@ export var dxItem: any;
  * @namespace DevExpress.ui
  * @public
  */
-export type format = 'billions' | 'currency' | 'day' | 'decimal' | 'exponential' | 'fixedPoint' | 'largeNumber' | 'longDate' | 'longTime' | 'millions' | 'millisecond' | 'month' | 'monthAndDay' | 'monthAndYear' | 'percent' | 'quarter' | 'quarterAndYear' | 'shortDate' | 'shortTime' | 'thousands' | 'trillions' | 'year' | 'dayOfWeek' | 'hour' | 'longDateLongTime' | 'minute' | 'second' | 'shortDateShortTime' | string | ((value: number | Date) => string) | {
+export type format = 'billions' | 'currency' | 'day' | 'decimal' | 'exponential' | 'fixedPoint' | 'largeNumber' | 'longDate' | 'longTime' | 'millions' | 'millisecond' | 'month' | 'monthAndDay' | 'monthAndYear' | 'percent' | 'quarter' | 'quarterAndYear' | 'shortDate' | 'shortTime' | 'thousands' | 'trillions' | 'year' | 'dayOfWeek' | 'hour' | 'longDateLongTime' | 'minute' | 'second' | 'shortDateShortTime' | string | ((value: number | Date) => string) | any | {
```

Comment:
Here I've added `any` since we support Intl and Globalize formats according to [the doc](https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/format/)

Reply:
tbd: Record<string, any>

Reply:
see js\core\utils\deferred.d.ts (exports dxpromise...)
public type should start with capital case

Reply:
import { Format } from '...'
//deprecated <-- TODO - check how to use it.
export type format = Format;